schedules_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var SchedulesHttpMock HttpClient = &HttpMock{
```

Comment:
Codacy found an issue: [var SchedulesHttpMock should be SchedulesHTTPMock](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219)